inc/namespace.php

```diff
@@ -1204,6 +1210,39 @@ function enhance_user_search_query( array $query, array $args ) : array {
 	return enhance_search_query( $query, $args, 'user' );
 }
 
+/**
+ * Use date decay settings from Altis config, if specified.
+ *
+ * This function retrieves each of the decay function's parameters.
+ *
+ * @var string "offset"     How far in the past a post must be before it starts to decay.
+ * @var string "scale"      The reference interval of the date distribution
+ * @var float  "decay"      Factor by which relevance decays at each interval from the offset.
+ * @var sting  "boost_mode" How this score of this function affects overall score.
+ *
+ * @uses $wp_filter global, for detecting the current filter.
+ * @param string|float $default_value Current value for the value being filtered.
+ * @return string|float Updated value.
+ */
+function apply_date_decay_config_values( $default_value ) {
+
+	// Ensure this is running on a filter for one of the allowed list of fields.
+	$matched_filter = preg_match(
+		'~epwr_(scale|decay|offset|boost_mode)~',
```

Comment:
Feels a bit magical! I'll allow it though.

Reply:
![](https://media1.tenor.com/images/dc8eb353369837ea36b6beddc21d8801/tenor.gif?itemid=5359025)